software/controller/controller.sh

```diff
@@ -131,7 +131,7 @@ install_linux() {
 
 configure_platformio() {
   echo 'ATTRS{idVendor}=="0483", ATTRS{idProduct}=="374b", MODE="666"' | sudo tee /etc/udev/rules.d/99-openocd.rules > /dev/null
-  curl -fsSL https://raw.githubusercontent.com/platformio/platformio-core/master/scripts/99-platformio-udev.rules | sudo tee /etc/udev/rules.d/99-platformio-udev.rules
+  curl -fsSL https://raw.githubusercontent.com/platformio/platformio-core/develop/platformio/assets/system/99-platformio-udev.rules | sudo tee /etc/udev/rules.d/99-platformio-udev.rules
```

Comment:
Are we certain we want to use develop branch?

Reply:
This is what they currently recommend in their docs:
https://docs.platformio.org/en/latest/core/installation/udev-rules.html